lib/desired-caps.js

```diff
@@ -0,0 +1,14 @@
+let geckodriverConstraints = {
+  geckodriverPort: {
+    isNumber: true
+  },
+  // duplicate of above with better spelling
+  geckoDriverPort: {
+    isNumber: true
+  },
+  firefoxInitialUrl: {
+    isString: true
+  }
+};
```

Comment:
make sure the noReset option is supported

Reply:
@mykola-mokhnach doesnt seem geckodriver is supporting this as it's always running `adb shell pm:clear` as default
https://hg.mozilla.org/mozilla-central/file/tip/testing/geckodriver/src/android.rs#l223
Im going to understand and try to create a PR on geckodriver

Reply:
How about [--connect-existing](https://firefox-source-docs.mozilla.org/testing/geckodriver/Flags.html#connect-existing) and [--profile](https://developer.mozilla.org/en-US/docs/Web/WebDriver/Capabilities/firefoxOptions)?